source/admin_manual/obox/index.rst

```diff
@@ -173,6 +173,19 @@ It's also possible to flush only indexes with specified priority (and below) wit
 
 If a user no longer actually exists on filesystem, it can be removed from metacache process with:
 
+
+.. code-block:: none
```

Comment:
Move this newly added text after the "doveadm metacache remove user@domain" below.